src/include/liburing.h

```diff
@@ -19,6 +19,17 @@
 #include "liburing/io_uring_version.h"
 #include "liburing/barrier.h"
 
+#if defined(__has_feature)
+#if __has_feature(address_sanitizer) && !defined(__SANITIZE_ADDRESS__) // for some clang versions
+#define __SANITIZE_ADDRESS__
```

Comment:
Should this perhaps be called __LIBURING_SANITIZE_ADDRESS__ or something like that? Just to keep the name scope clean.